bin/plot_krun_results

```diff
@@ -126,13 +135,16 @@ def main(is_interactive, data_dcts, plot_titles, window_size, outfile,
     # a flat list of all run sequences.
     if one_page:
         # XXX revise this
```

Comment:
I think it was because in an earlier change I used multi-page mode first, and came back to single page later.